### ISSUE TITLE
Keyboard Focus is not clearly visible on cancel button: A11y_Visual Studio Code Web Extensions_Github Pull request_Create Pull request_Keyboard

### ISSUE THREAD

**Issue by Rama8345** [microsoft/vscode-pull-request-github#6449]
### GIT Hub Tags:
#A11yTCS; #A11ySev2; #Visual Studio Code Web Extensions; #BM_Visual Studio Code Web Extensions_Web_Oct2024; #DesktopWeb; #A11yMAS; #WCAG2.4.7; #FocusVisible; #AILimited; #Win11; #FTP; #Chrome;

### Environment and OS details:
Application name: Visual Studio Code Web Extensions
Chrome Version: Version 130.0.6723.92 (Official Build) (64-bit)
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github

### Reproduction Steps:

1. Open url https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github environment and login with git hub account.
2. TAB to 'Github Pull request' and press enter.
3. TAB to 'Create Pull request' and press enter. 
4. TAB to "Cancel" button.
5. Observe the issue whether Keyboard Focus is clearly visible on cancel button or not.

### Actual behavior:
Keyboard Focus is not clearly visible on cancel button.

**Similar issue is observed for all buttons.**

### Expected behavior:
Keyboard Focus should be clearly visible on cancel button.

### User Impact:
Low vision users will not be able to understand where the keyboard focus is present if Keyboard Focus is not clearly visible on cancel button.

### Attachment:
![Image](https://github.com/user-attachments/assets/03c4f76c-8427-4027-8862-8d90281cb2ac)

**Comment by PoojaNamde** [comment]
#Closed; #A11yFixed;

Verified the issue in the env, issue is fixed hence closing the bug.

<img width="1359" height="678" alt="Image" src="https://github.com/user-attachments/assets/f4795ac8-0ffa-4647-a1eb-37a90ec54f08" />
